packages/test/test-service-load/src/workloads/gc/testConfig.json

```diff
@@ -4,42 +4,40 @@
 			"opRatePerMin": 600,
 			"progressIntervalMs": 5000,
 			"numClients": 5,
-			"totalSendCount": 300,
+			"totalSendCount": 1200,
```

Comment:
Why?  And the fault injection one below.

Reply:
It used to be 1200 but I changed it to 300 by accidentally pushing it with what I was testing locally with.

With 300, the test will run for ~30 seconds (300 / 600 for opRatePerMin). Wit sweep timeout of 40 seconds, nothing will be swept or tombstoned. With 1200, it will run for ~2 mins.

Reply:
I changed faultInjection max because with 20 seconds, at latest, a fault will happen before or the same time as session expiry which is also 20 seconds. So, session expiry will not get to run at all. With 60 seconds, both have a chance to run.